src/store/bookingPortal/index.js

```diff
@@ -29,6 +38,21 @@ const actions = {
         } else if (res.DisplayMsg) {
             commit('set-error', res.DisplayMsg + ' ( ' + res.ErrorMsg + ' )');
         }
+        commit('set-loading', false);
+    },
+    async getPaymentLink({ dispatch, commit }, reqBody) {
+        commit('set-loading', true);
+        const res = await mayaClient.post(
+            '/payment/generate-payment-link',
+            reqBody,
+        );
+        if (res) {
```

Comment:
if there is an error maya will still send res with DisplayMsg but without any paymentDetails
So this condition will still evaluate true
Else condition will never be evaluated